.github/ISSUE_TEMPLATE/10-debugging-experience.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: "Internal: Feedback on the CI, Test Frameworks and Debugability"
```

Comment:
```suggestion
name: "Internal: Feedback on CI, Test Frameworks and Debuggability"
```

Reply:
This doesn't look like it got addressed!

Reply:
Yes sorry I forgot to force-push. It was all addressed.